README.md

```diff
@@ -1,18 +1,18 @@
-# Blox Eth Key Manager
+# Eth Key Manager
 
 <!-- [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fbloxapp%2Feth2-key-manager.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fbloxapp%2Feth2-key-manager?ref=badge_shield) -->
```

Comment:
the FOSSA badge is currently commented out.
Do we still need FOSSA integration for this project, and if so, who has access to manage it? If it’s no longer in use, we could consider removing the reference altogether to keep things clean

@moshe-blox @y0sher

Reply:
Talked with @moshe-blox seems to be added to run some sort of security checks in the past, doesn't seem to be relevant anymore.
Removing.